pkg/agent/openflow/framework.go

```diff
@@ -213,7 +213,7 @@ func (f *featureNetworkPolicy) getRequiredTables() []*Table {
 			AntreaPolicyEgressRuleTable,
 			AntreaPolicyIngressRuleTable,
 		)
-		if f.nodeType == config.K8sNode {
```

Comment:
I think you can only use condition "f.enableL7NetworkPolicy", and disable feature "L7NetworkPolicy" on VM Agent cases.

Reply:
Done